lib/tasks/debug.rake

```diff
@@ -75,7 +75,7 @@ namespace :debug do
 
       maxlen = results[:query_scores].map { |query, _| query.length }.max
       results[:query_scores].each do |query, score|
-        puts "#{(query + ':').ljust(maxlen + 1)} #{score}"
+        puts "#{("#{query}:").ljust(maxlen + 1)} #{score}"
```

Comment:
This is a bit of nitpick, but the nested `"` look a bit odd. Can they be replaced with single-quotes?
```suggestion
        puts "#{('#{query}:').ljust(maxlen + 1)} #{score}"
```

Reply:
Thanks for the suggestion here, I'd agree that this looks a bit odd. 

I felt that the underlying code was already rather cryptic so I've gone with an approach of breaking that out instead in https://github.com/alphagov/search-api/pull/2266/commits/97f56af45cbc3e27796279ded2ed4132f4e904e1

Reply:
Splitting them out is better.